types/mailgun-js/index.d.ts

```diff
@@ -149,6 +160,23 @@ declare namespace Mailgun {
             address: string,
             callback?: (error: Error, body: validation.ValidateResponse) => void
         ): Promise<validation.ValidateResponse>;
+        validate(
+            address: string,
+            isPrivate: boolean,
+            callback?: (error: Error, body: validation.ValidateResponse) => void
+        ): Promise<validation.ValidateResponse>;
+        validate(
+            address: string,
+            // tslint:disable-next-line unified-signatures
+            opts: { [key: string]: any },
```

Comment:
Is `opts: object,`sufficient here instead?

Reply:
Since the mailgun docs specify exactly which parameters may be used, I will adjust `opts` accordingly.